### ISSUE TITLE
How can I call verify documents of users

### ISSUE THREAD

**Issue by bacancy-gautam** [fundamerica/trulioo#2]
Hello, I want to verify user's documents via this gem
How can I do that?
Can any one help me which steps I have to do in my ruby on rails application?

**Comment by mangukiya-ketan** [comment]
Yes, i am also facing the same issue, Can any one please Help us to out from this issue?
